torchvision/utils.py

```diff
@@ -179,12 +183,21 @@ def draw_bounding_boxes(
 
     img_boxes = boxes.to(torch.int64).tolist()
 
-    draw = ImageDraw.Draw(img_to_draw)
+    if fill:
+        draw = ImageDraw.Draw(img_to_draw, "RGBA")
+        fill = colors + [100]
```

Comment:
I'm keeping `A` channel value fixed to 100. Otherwise we would have to make user pass `Tuple[int, int, int, int]`